test/.kontena_client.json

```diff
@@ -0,0 +1,24 @@
+{
+  "current_server": "compose-e2e",
+  "current_account": "kontena",
+  "servers": [
+    {
+      "name": "compose-e2e",
+      "url": "http://localhost:9292",
+      "username": "admin",
+      "grid": "e2e",
+      "token": "50f480e2c6a62dc54ba4aeb219c447329e5634a5e9fb9582e994d94ae113ac16",
+      "token_expires_at": 0,
+      "refresh_token": null
+    }
+  ],
+  "accounts": [
+    {
+      "name": "kontena",
+      "username": "pabloduo",
+      "token": "0bd4d4155c364de79ef264089a3e8d0f",
```

Comment:
oops?

Reply:
@SpComb yes... I should exercise extreme caution and not push changes to this file 😃 thanks for the catch - I've already revoked this token and confirmed both `token` and `refresh_token` are no longer authorized to access my Kontena Cloud account.